client/src/images/undraw_composition_oskp.svg

```diff
@@ -0,0 +1 @@
861.90745,469.17766,873.279,487.7426Z" transform="translate(-68 -79)" fill="#3f3d56"/><path d="M906.26918,689.59119l-1.65722-1.12011A51.9189,51.9189,0,0,0,913.775,659.6786c-.04639-15.05762-7.44238-28.98487-14.59521-42.45362-1.32764-2.5-2.65088-4.99218-3.92383-7.48242A163.33418,163.33418,0,0,1,882.5519,577.85c-11.09473-41.23-.17139-89.87353,29.21923-130.12207,23.31543-31.92871,58.07178-59.84863,106.25391-85.35547l.93555,1.76758c-47.90772,25.36133-82.44141,53.08936-105.57422,84.76758-29.03711,39.76367-39.8418,87.77246-28.90283,128.42285a161.39429,161.39429,0,0,0,12.55273,31.50195c1.269,2.481,2.5874,4.96338,3.91016,7.45459,7.2666,13.6836,14.78076,27.833,14.82861,43.38575A53.937,53.937,0,0,1,906.26918,689.59119Z" transform="translate(-68 -79)" fill="#3f3d56"/><circle cx="1044.99961" cy="267.00039" r="19" fill="#3f3d56"/><path d="M1068,111H68V79H1068ZM70,109h996V81H70Z" transform="translate(-68 -79)" fill="#3f3d56"/><path d="M90,105a10,10,0,1,1,10-10A10.01114,10.01114,0,0,1,90,105Zm0-18a8,8,0,1,0,8,8A8.00917,8.00917,0,0,0,90,87Z" transform="translate(-68 -79)" fill="#3f3d56"/><path d="M117,105a10,10,0,1,1,10-10A10.01114,10.01114,0,0,1,117,105Zm0-18a8,8,0,1,0,8,8A8.00917,8.00917,0,0,0,117,87Z" transform="translate(-68 -79)" fill="#3f3d56"/><path d="M144,105a10,10,0,1,1,10-10A10.01114,10.01114,0,0,1,144,105Zm0-18a8,8,0,1,0,8,8A8.00917,8.00917,0,0,0,144,87Z" transform="translate(-68 -79)" fill="#3f3d56"/><rect x="13.5" y="444" width="224.5" height="41" fill="#6c63ff"/><path d="M322.5,565H80.5V510h242Zm-240-2h238V512H82.5Z" transform="translate(-68 -79)" fill="#3f3d56"/><path d="M227.35913,234.19344c-5.68507,21.01543-25.15771,34.03081-25.15771,34.03081s-10.25532-21.05736-4.57026-42.07279,25.15772-34.03081,25.15772-34.03081S233.0442,213.178,227.35913,234.19344Z" transform="translate(-68 -79)" fill="#6c63ff"/><path d="M175.64087,234.19344c5.68507,21.01543,25.15771,34.03081,25.15771,34.03081s10.25532-21.05736,4.57026-42.07279-25.15772-34.03081-25.15772-34.03081S169.9558,213.178,175.64087,234.19344Z" transform="translate(-68 -79)" fill="#6c63ff"/><path d="M183.99467,229.677C199.59458,244.86288,201.5,268.20712,201.5,268.20712s-23.387-1.27695-38.987-16.46283-17.50532-38.53012-17.50532-38.53012S168.39476,214.49111,183.99467,229.677Z" transform="translate(-68 -79)" fill="#3f3d56"/><path d="M219.00533,229.677C203.40542,244.86288,201.5,268.20712,201.5,268.20712s23.387-1.27695,38.987-16.46283,17.50532-38.53012,17.50532-38.53012S234.60524,214.49111,219.00533,229.677Z" transform="translate(-68 -79)" fill="#3f3d56"/><rect x="49.5" y="76.5" width="168" height="10" fill="#3f3d56"/><rect x="49.5" y="226.5" width="168" height="10" fill="#3f3d56"/><rect x="49.5" y="248.5" width="168" height="10" fill="#3f3d56"/><rect x="49.5" y="270.5" width="168" height="10" fill="#3f3d56"/><rect x="782.5" y="76.5" width="168" height="10" fill="#3f3d56"/><rect x="782.5" y="98.5" width="168" height="10" fill="#3f3d56"/><rect x="782.5" y="120.5" width="168" height="10" fill="#3f3d56"/></svg>
```

Comment:
What is this for?

Reply:
uh oh I was going to use it for 404 not found page, but then ended up not using it.